tests/components/wsdot/test_sensor.py

```diff
@@ -23,42 +20,50 @@
 
 config = {
     CONF_API_KEY: "foo",
-    SCAN_INTERVAL: timedelta(seconds=120),
     CONF_TRAVEL_TIMES: [{CONF_ID: 96, CONF_NAME: "I90 EB"}],
 }
 
 
+@pytest.fixture
+def test_travel_time() -> wsdot.TravelTime:
+    """Return TravelTime data based on test fixture payload."""
+    test_data = load_fixture("wsdot/wsdot.json")
+    test_response = json.loads(test_data)
+    return wsdot.TravelTime(**test_response)
```

Comment:
Instead, create a `conftest.py` where we `patch("homeassistant.components.wsdot.sensor.WsdotTravelTimes")` and then set the functions to return what we expect